Makefile

```diff
@@ -3,10 +3,8 @@
 # Project variables.
 PROJECT_NAME = starport
 DATE := $(shell date '+%Y-%m-%dT%H:%M:%S')
-VERSION = development
```

Comment:
we can use the version generated locally based on the git tag and commit:

```shell
VERSION := $(shell git describe --tags --always)
```

Reply:
this was the case before but you need to fetch the tags first which takes a long time. but I don't think there is an advantage of having the tag info in the development builds since we already know the hash.